contracts/governance/timelock-controller/tests/tests.rs

```diff
@@ -180,7 +184,7 @@ mod tests {
     }
 
     fn setup() -> DefaultAccounts<DefaultEnvironment> {
-        let accounts = ink_env::test::default_accounts::<ink_env::DefaultEnvironment>().expect("Cannot get accounts");
+        let accounts = accounts();
 
         accounts
```

Comment:
```suggestion
        accounts()
```

Reply:
`accounts` variable is used in the test=)

Reply:
right 😄